controllers/storagecluster/generate.go

```diff
@@ -2,11 +2,19 @@ package storagecluster
 
 import (
 	"fmt"
+	"math"
+	"strconv"
 
 	ocsv1 "github.com/red-hat-storage/ocs-operator/api/v1"
 	cephv1 "github.com/rook/rook/pkg/apis/ceph.rook.io/v1"
 )
 
+const (
+	StorageDeviceName = "default"
```

Comment:
Why do we need this constant? it has no meaning outside searching for the backing pool in `generateCephFSProviderParameters`, and the name does not imply anything about provide mode or Ceph fs (the name lack any kind of context). 

Let's just use the actual string in the search operation instead of a constant. Any future change will be made in a single location in both cases, and for someone reading the search code, all information will be stated locally (no need to jump to the top of the file and back again to the code while reading)